src/Features/CSharp/Portable/RemoveUnusedParametersAndValues/CSharpRemoveUnusedParametersAndValuesCodeFixProvider.cs

```diff
@@ -0,0 +1,187 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Composition;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.RemoveUnusedParametersAndValues;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.RemoveUnusedParametersAndValues
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.RemoveUnusedParametersAndValues), Shared]
+    internal class CSharpRemoveUnusedParametersAndValuesCodeFixProvider:
```

Comment:
note: i'm slightly surprised this is one fixer.  i wouldn't have been surprised to find a fixer for parameters, and a fixer for values.

Reply:
Thanks, I forgot to remove the "Parameters" part from the name. Fixed now.

---
In reply to: [230603336](https://github.com/dotnet/roslyn/pull/30777#discussion_r230603336) [](ancestors = 230603336)